helper/funcs.go

```diff
@@ -343,10 +344,12 @@ func CopySliceInt(s []int) []int {
 
 // CleanEnvVar replaces all occurrences of illegal characters in an environment
 // variable with the specified byte.
-func CleanEnvVar(s string, r byte) string {
+// any extra characters will be considered valid
+func CleanEnvVar(s string, r byte, e string) string {
```

Comment:
What about `replacement byte` and `except string`? with the additional parameter, I found myself expecting string replacement param order.